src/twisted/names/dns.py

```diff
@@ -1902,6 +1917,79 @@ def __hash__(self):
 
 
 
+@implementer(IEncodable, IRecord)
+class Record_SSHFP(tputil.FancyEqMixin, tputil.FancyStrMixin):
+    """
+    A record containing the fingerprint of an SSH key.
+
+    @type keytype: L{int}
+    @ivar keytype: The SSH key algorithm or key type.
```

Comment:
key algorithm or key type... which one?

Looking at the RFC, I think that this is referred as "algorithm number" and I would prefer to have that terminology here

Reply:
The user documentation (e.g. the ssh man pages) mostly refer to it as the "key type" but the RFCs (RFC4255, RFC4252) seem to prefer "algorithm". I'll change it to `algorithm` and `ALGORITHM_RSA` etc. There's the possibility of different key types using the same underlying algorithm but that seems unlikely to confuse people in practice.